server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java

```diff
@@ -159,6 +163,17 @@ && getCatalogDAO(session, updateCatalog.getNewName()) != null) {
         if (updateCatalog.getComment() != null) {
           catalogInfoDAO.setComment(updateCatalog.getComment());
         }
+        if (!updateCatalog.getProperties().isEmpty()) {
```

Comment:
Please use both `null` and `isEmpty()` checks for properties' absence for consistency.